hub/views/landingpages.py

```diff
@@ -0,0 +1,40 @@
+from django.views.generic import TemplateView
+
+from hub.mixins import TitleMixin
+
+class BaseLandingPageView(TitleMixin, TemplateView):
+    template_name = "hub/landing_page.html"
+
+    # Default values which could be overridden in the extending Views
+    example_filter_label = "Great Big Green Week events 2023"
+    example_filter_comparator = "is greater than"
+    example_filter_value = "1"
+    example_shader_label = "MP majority"
+    secondary_text = "The Local Intelligence Hub was developed by The Climate Coalition in partnership with mySociety and Green Alliance. Anyone can access our free and growing library of live data about local MPs, political candidates, constituencies, and people and groups who are already taking action on the climate and nature crises. The tool also brings together recent public opinion data on climate action and the cost of living crisis, the climate and nature movement so people can explore how their local area compares to the national average."
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context["title"] = self.page_title
+        context["meta_description"] = self.meta_description
+        context["example_filter_label"] = self.example_filter_label
+        context["example_filter_comparator"] = self.example_filter_comparator
+        context["example_filter_value"] = self.example_filter_value
+        context["example_shader_label"] = self.example_shader_label
+        context["secondary_text"] = self.secondary_text
+        context["intro_text"] = self.intro_text
+        context["search_form_label"] = self.search_form_label
+        return context
+
+
+class LandingPage01View(BaseLandingPageView):
+    page_title = "Climate data for your area"
+    meta_description = "" # TODO
+    intro_text = "How much support is there for climate action in your local community? How did your MP vote on environmental issues? What other factors affect climate action in your constituency?"
+    search_form_label = "Input your postcode to see all the climate-related data for your area."
+
+
+class LandingPage02View(BaseLandingPageView):
+    page_title = "Public opinion on climate"
+    meta_description = "" # TODO
+    intro_text = "If you're campaigning for climate or enviornment issues, it's useful to know how much support there is for decarbonisation initiatives in your local area. Who backs climate action, and what percentage of the population backs net zero?"
```

Comment:
Typo in environmental. Hello!
```suggestion
    intro_text = "If you're campaigning for climate or enviromental issues, it's useful to know how much support there is for decarbonisation initiatives in your local area. Who backs climate action, and what percentage of the population backs net zero?"
```